### PR TITLE
Changed the homepage picture and the Navbar logo.

### PR DESCRIPTION
**Pull request by agazielly**
@ team,

I did some changes on the homepage on the picture and also on the logo of the Navbar. Let me know what you think.

Pending tasks on the homepage:

- [x] Implement the carousel

- [ ] Make the navbar transparent.

### CODE REVIEW

File: app/views/shared/_navbar.html.erb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div class="navbar navbar-expand-sm navbar-light navbar-lewagon">
   <%= link_to "#", class: "navbar-brand" do %>
-    <%= image_tag "https://raw.githubusercontent.com/lewagon/fullstack-images/master/uikit/logo.png" %>
+    <%= image_tag "https://res.cloudinary.com/agazielly/image/upload/v1566246427/joan-mm-EHuyA_oUCok-unsplash_ymnzy9.jpg" %>
```
Comment (AlexandreCoulier) [inline app/views/shared/_navbar.html.erb]: Is it a random picture ?

Comment (agazielly) [reply app/views/shared/_navbar.html.erb]: No. I chose a picture from Unsplash website. I wanted it fixed.

**Comment by agazielly** [comment]
I still need to do some improvement on the navbar but I suggest to deploy it for the moment.

File: app/views/shared/_navbar.html.erb (hunk 1 above)
Comment (AlexandreCoulier) [reply app/views/shared/_navbar.html.erb]: ok !

**Pull request merged**